### ISSUE TITLE
mobile scrolling issues

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#8673]
@mbastiaan reported scrolling issues in #8464:

> Sometimes it's impossible to scroll a page up/down, closing browser the tab and reopening Cockpit solves the problem (not sure if this is actually a Cockpit problem, but it's the only website I have this problem with).

Which pages are affected?

> All pages. Right now I just went from System to Network to Storage to System and none of the pages are scrolling.

**Comment by garrett** [comment]
Hi, @mbastiaan:

It's been a while since you reported scrolling issues in Cockpit in Firefox on your Samsung phone (#8464).

I haven't been able to reproduce it on my own Android phones running Firefox (Google Nexus 5 & 6p, OnePlus 6T).

Could you let us know if it was solved with some of the fixes that landed recently or if you still have issues scrolling the page?

Thanks!

**Comment by mbastiaan** [comment]
Hi @garrett,

I've been able to test it again on Fedora 29 with version 185 and the problem still persists.
Version 186 isn't available yet as an update, any idea when it should be available?

**Comment by KKoukiou** [comment]
@mbastiaan there is already 187 is fedora 29. Indeed we did a lot of enhancements for mobile accessibility in the latest releases so definitely try!

**Comment by garrett** [comment]
@KKoukiou: I think this is related to some glitch causing scrolling to not work on Firefox on a Samsung phone, not related to layouts.

I haven't been able to reproduce on my own phones (& tablet) running Google stock Android, LineageOS, and OxygenOS... on either Firefox or Chrome. But I'll try again.

However, a _lot_ of people use Samsung devices ([20% of global phone usage, which is the # 1 spot by far — almost double of Apple's iOS](https://www.idc.com/promo/smartphone-market-share/vendor)), so it's important that Cockpit works on them.

**Comment by marusak** [comment]
I have booted up my ancient phone (well ancient, I was still using it as my only phone a year ago :D) Samsung Galaxy Ace 2 from 2012 with Android 4.1.2 Jelly Bean and Firefox (installed today what was available on the market). Even though the phone is super slow I was able to load cockpit and scroll through pages just fine.

So I am afraid we cannot reproduce this not even on older samsung phones with firefox. Closing as insufficient data.
